NotificationsActivity Crash

Just one report of this so far:

java.lang.RuntimeException: Unable to start activity ComponentInfo{org.wordpress.android/org.wordpress.android.ui.notifications.NotificationsActivity}: java.lang.NullPointerException
at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2180)
at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:2230)
at android.app.ActivityThread.access$600(ActivityThread.java:141)
at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1234)
at android.os.Handler.dispatchMessage(Handler.java:99)
at android.os.Looper.loop(Looper.java:137)
at android.app.ActivityThread.main(ActivityThread.java:5039)
at java.lang.reflect.Method.invokeNative(Native Method)
at java.lang.reflect.Method.invoke(Method.java:511)
at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:793)
at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:560)
at dalvik.system.NativeStart.main(Native Method)
Caused by: java.lang.NullPointerException
at org.wordpress.android.ui.notifications.NotificationsListFragment$NotesAdapter.addAll(NotificationsListFragment.java:143)
at org.wordpress.android.ui.notifications.NotificationsActivity.onCreate(NotificationsActivity.java:79)
at android.app.Activity.performCreate(Activity.java:5104)
at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1080)
at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2144)
... 11 more

--- a/src/org/wordpress/android/ui/notifications/NotificationsListFragment.java
+++ b/src/org/wordpress/android/ui/notifications/NotificationsListFragment.java
@@ -140,7 +140,7 @@ public class NotificationsListFragment extends ListFragment {
             if (notes.size() == 0) {
                 // No more notes available
                 mAllNotesLoaded = true;
-                if(mPressFooterView != null)
+                if(mProgressFooterView != null)
                     mProgressFooterView.setVisibility(View.GONE);
             } else {
                 Iterator<Note> noteIterator = notes.iterator();

